.env.example

```diff
@@ -103,3 +103,12 @@ ARKSCAN_WEBHOOKS_TRANSACTION_APPLIED_TTL=8
 ARKSCAN_WEBHOOKS_WALLET_VOTE_TTL=8
 ARKSCAN_WEBHOOKS_CURRENCY_UPDATE_TTL=8
 ARKSCAN_WEBHOOKS_STATISTICS_UPDATE_TTL=8
+
+ARKSCAN_CONTRACT_CONSENSUS_ADDRESS=
+
+ARKSCAN_CONTRACT_TRANSFER_METHOD=a9059cbb
+ARKSCAN_CONTRACT_BATCH_TRANSFER_METHOD=88d695b2
+ARKSCAN_CONTRACT_VALIDATOR_REGISTRATION_METHOD=602a9eee
+ARKSCAN_CONTRACT_VOTE_METHOD=6dd7d8ea
+ARKSCAN_CONTRACT_UNVOTE_METHOD=3174b689
+ARKSCAN_CONTRACT_VALIDATOR_RESIGNATION_METHOD=b85f5da2
```

Comment:
would leave it out as we already define them in the config file per network and will not be adjusting them for official networks

Reply:
I've kept the keys but removed the values